python/kwiver/vital/tests/CMakeLists.txt

```diff
@@ -43,9 +43,28 @@ set(__all_modnames
   ${__testable_modnames}
   )
 
-# Add files to the vital.tests python module
+
+set(mod_dst)
 foreach (modname IN LISTS __all_modnames)
   kwiver_add_python_module( ${CMAKE_CURRENT_SOURCE_DIR}/${modname}.py
     vital/tests
     ${modname})
 endforeach()
+
+
+if (KWIVER_ENABLE_TESTS)
+
```

Comment:
I don't think you need to predefine things that are empty like NOSE_RUNNER, kwiver_test_working_path, kwiver_test_working_path. I could be wrong depending on context but generally it's not required.

*Edit ... in the event that you need to explicitly set these so you can set up correct behaviour in kwiver-setup-tests-python, then I would suggest that you should revisit the design here. Best to avoid required side-effects.

Reply:
Those definitions were/are unnecessary, they will be removed.